### PR TITLE
[JEX-608] Use Expeditor to bump version and trigger release build

### PR DESCRIPTION
**Pull request by schisamo**
This change updates this repo to begin using the following standard
behavior from Expeditor following a merge to the master branch:

* Automatically bump the `PATCH` version in the `VERSION` file (see https://git.io/vQQYR)
* Create a git tag for the new version (see https://git.io/vQQYR)
* Trigger a release build on manhattan.ci.chef.co using the new tag (see https://git.io/vQQY1)
* Automatically update CHANGELOG (coming soon...the `built_in:update_changelog` merge action is in development)

More details on Expeditor can be found at:
https://github.com/chef/es-lita/tree/master/lita-expeditor

Signed-off-by: Seth Chisamore <schisamo@chef.io>

### CODE REVIEW

File: .expeditor/update_version.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+# Temporarily fall back to cat-ing version since this change isn't in es-lita yet.
```
Comment (tduffield) [inline .expeditor/update_version.sh]: I'm actually removing this functionality. We need to cat.

Comment (schisamo) [inline .expeditor/update_version.sh]: @tduffield So are you saying to just remove this comment?

Comment (tduffield) [inline .expeditor/update_version.sh]: sorry, not. the line should be `sed -i -r "s/VERSION = \".+\"/VERSION = \"$(cat VERSION)\"/" lib/push_client/version.rb

**Comment by schisamo** [comment]
I'll merge this once we have a new release of Expeditor with CHANGELOG support fully deployed.

**Pull request merged**